server/src/main/java/com/datastrato/gravitino/server/web/rest/CatalogOperations.java

```diff
@@ -71,10 +75,15 @@ public Response listCatalogs(
                 () -> {
                   if (verbose) {
                     Catalog[] catalogs = catalogDispatcher.listCatalogsInfo(catalogNS);
-                    return Utils.ok(new CatalogListResponse(DTOConverters.toDTOs(catalogs)));
+                    Response response =
+                        Utils.ok(new CatalogListResponse(DTOConverters.toDTOs(catalogs)));
+                    LOG.info("list {} catalogs info under metalake: {}", catalogs.length, metalake);
+                    return response;
                   } else {
                     NameIdentifier[] idents = catalogDispatcher.listCatalogs(catalogNS);
-                    return Utils.ok(new EntityListResponse(idents));
+                    Response response = Utils.ok(new EntityListResponse(idents));
+                    LOG.info("list {} catalogs under metalake: {}", idents.length, metalake);
```

Comment:
Can you please capitalize the first letter for all the sentense.

Reply:
fixed